src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="info">
+        <legend>Personal information</legend>
+        <div class="info__data">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="info__data">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="info__data">
+          <label for="age">How old are you?</label>
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="info__data">
+          <label for="birthday">Full date of birth:</label>
+          <input
+            type="date"
+            name="birthday"
+            id="birthday"
+          >
+        </div>
+        <div class="info__data">
+          <label for="terms">I accept the terms of agreement</label>
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="info">
+        <legend>Registration</legend>
+        <div class="info__data">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="info__data">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="info">
+        <legend>An interesting fact about you!</legend>
+        <div class="info__data">
+          <label for="catLover">Do you love cats?</label>
+          <input
+            type="radio"
+            name="catLover"
+            id="catLover"
+            value="Yes"
+          >Yes
+          <!-- maybe should add required attribute to specifies that an input field must be filled out -->
+          <input
+            type="radio"
+            name="catLover"
+            id="catLover1"
+            value="No"
+          >No
```

Comment:
Use label for this inputs so that when you click on the word radio button become selected